app/controllers/api/v1/merchants_controller.rb

```diff
@@ -0,0 +1,33 @@
+# app/controllers/api/v1/merchants_controller.rb
+
+class Api::V1::MerchantsController < ApplicationController
+
+  def index
+    render json: MerchantSerializer.new(Merchant.all)
+  end
+
+  def show
+    render json: MerchantSerializer.new(Merchant.find(params[:id]))
+  end
+
+  def create
+    render json: MerchantSerializer.new(Merchant.create(merchant_params))
+  end
+
+  def update
+    render json: MerchantSerializer.new(Merchant.update(params[:id], merchant_params))
+  end
+
+  def destroy
+    render json: MerchantSerializer.new(Merchant.find(params[:id]))
+    Merchant.delete(params[:id])
```

Comment:
@BrianZanti @dionew1 Is there a more economical way to render the deleted resource with its attributes?

Reply:
No, this is how I would render that resource with it's attribute.